homeassistant/components/command_line/switch.py

```diff
@@ -65,6 +68,10 @@ async def async_setup_platform(
     switches = []
 
     for object_id, device_config in devices.items():
+        _config = {CONF_UNIQUE_ID: device_config.get(CONF_UNIQUE_ID)}
```

Comment:
maybe name this trigger_entity_config or something so it's clear what it is?
Also, I think you can add all the keys here instead of adding them one by one below.